staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -385,6 +385,19 @@ func GetGaugeMetricValue(m metrics.GaugeMetric) (float64, error) {
 	return metricProto.Gauge.GetValue(), nil
 }
 
+// GetCounterMetricValueWithLabels returns float64 value for the counter metric filtered by the label value mapping
+func GetCounterMetricValueWithLabels(counterVec *metrics.CounterVec, lvMap map[string]string) (float64, error) {
+	counter, err := counterVec.GetMetricWith(lvMap)
```

Comment:
you can use `counterVec.WithLabelValues` instead.

Reply:
Done